changelog/fragments/helm-install-status-fix.yaml

```
@@ -0,0 +1,4 @@
+entries:
+  - description: Fixed a bug that caused the Helm operator not to set the `InstallSuccessful` status.
```

```suggestion
  - description: Ensures that the `InstallSuccessful` status will be set always by checking if the release info. 
```

---

IHMO ^

---

I think @mikeshng's version reads more clearly with one tweak though.
```suggestion
  - description: Fixed a bug that caused the Helm operator not to set the `InstallSuccessful` and `UpgradeSuccessful` status reasons when the status update fails during installation and upgrade.
```

---

i liked your latest version 👍